pkg/storage/snapshot/snapshot.go

```diff
@@ -855,15 +858,21 @@ func updateVMSnapshotIndications(source snapshotSource) ([]snapshotv1.Indication
 	return indications, nil
 }
 
-func (ctrl *VMSnapshotController) updateSnapshotSnapshotableVolumes(snapshot *snapshotv1.VirtualMachineSnapshot, content *snapshotv1.VirtualMachineSnapshotContent) error {
+func (ctrl *VMSnapshotController) updateSnapshotSnapshotableVolumes(snapshot *snapshotv1.VirtualMachineSnapshot, content *snapshotv1.VirtualMachineSnapshotContent, source snapshotSource) error {
 	if content == nil {
 		return nil
 	}
 	vm := content.Spec.Source.VirtualMachine
 	if vm == nil || vm.Spec.Template == nil {
 		return nil
 	}
-	volumes, err := storageutils.GetVolumes(vm, ctrl.Client, storageutils.WithAllVolumes)
+
+	opts, err := getVolumeOptions(source)
```

Comment:
I think you should get all the volumes including the backend pvc. in case the snapshot is online then put it in the excluded volumes

Reply:
I remember having complications but I'll look into it.